servers/services/src/main/java/org/projectnessie/services/impl/NamespaceApiImpl.java

```diff
@@ -91,10 +93,15 @@ public Namespace createNamespace(NamespaceParams params)
           };
 
       Preconditions.checkArgument(!namespace.isEmpty(), "Namespace name must not be empty");
-      Put put = Put.of(ContentKey.of(namespace.getElements()), namespace);
+      Namespace nsWithProps = namespace;
+      if (null != properties.getPropertyUpdates()) {
```

Comment:
Would be nice to assert that there are no property-deletions in `properties`

Reply:
done